docs/embedding/sdk/questions.md

```diff
@@ -139,26 +139,169 @@ To customize the layout, use namespaced components within the `InteractiveQuesti
+> Advanced query editor that provides full access to question configuration. Includes filtering, aggregation, custom expressions, and joins.
+
+*Replaces deprecated `InteractiveQuestion.Notebook`*
+
+| Prop | Type | Description |
+|------|------|-------------|
+| onApply | () => void | Callback when changes are applied, defaults to empty function |
+
+#### `InteractiveQuestion.EditorButton`
+> Toggle button for showing/hiding the Editor interface.
+
+*Replaces deprecated `InteractiveQuestion.NotebookButton`*
+
+Uses [Mantine ActionIcon props](https://v6.mantine.dev/core/action-icon/) under the hood, as well as:
+
+| Prop | Type | Description |
+|------|------|-------------|
+| isOpen* | boolean | Whether the editor is currently open |
+| className | string | |
+| style | React.CSSProperties | |
+
+
+#### `InteractiveQuestion.QuestionVisualization`
+> The main visualization component that renders the question results as a chart, table, or other visualization type.
+
+| Prop | Type | Description |
+|------|------|-------------|
+| height | number \| string | Height for visualization |
+| width | number \| string | Width for visualization |
+| className | string | |
+| style | React.CSSProperties | |
+
+#### `InteractiveQuestion.QuestionSettings`
+> Settings panel for configuring visualization options like axes, colors, and formatting.
+
+No props. Uses question context for settings.
+
+#### `InteractiveQuestion.QuestionSettingsDropdown`
+> Dropdown button that contains the QuestionSettings component.
+
+Uses [Popover props](https://v6.mantine.dev/core/popover/?t=props) except `onClose` and `opened` under the hood, as well as:
+
+| Prop | Type | Description |
+|------|------|-------------|
+| height | CSSProperties["height"] | Height for the dropdown menu |
+| className | string | |
+| style | React.CSSProperties | |
+
+#### `InteractiveQuestion.ChartTypeSelector`
+> Detailed chart type selection interface with sensible and non-sensible visualization options.
+
+Uses [Mantine Stack props](https://v6.mantine.dev/core/stack/?t=props) under the hood, as well as:
+
+| Prop | Type | Description |
+|------|------|-------------|
+| onChange | (display: CardDisplayType) => void | Callback when visualization type changes |
+| className | string | |
+| style | React.CSSProperties | |
+
+#### `InteractiveQuestion.ChartTypeDropdown`
+> Dropdown for selecting the visualization type (bar chart, line chart, table, etc.). Automatically filters to show only compatible visualization types for the current data.
+
+Uses [Mantine Menu props](https://v6.mantine.dev/core/menu/?t=props) under the hood, as well as:
+
+| Prop | Type | Description |
+|------|------|-------------|
+| className | string | |
+| style | React.CSSProperties | |
+
+#### `InteractiveQuestion.SaveQuestionForm`
+> Form for saving a question, including title and description. When saved:
+> - For new questions: Calls `onCreate` from context
+> - For existing questions: Calls `onSave` from context
```

Comment:
Should we clarify that their InteractiveQuestion is what provides the context question, and `onCreate` and `onSave` in this case comes from their InteractiveQuestion prop?